### ISSUE TITLE
Meaningless values?

### ISSUE THREAD

**Issue by PierreScerri** [gurbyz/power-wheel-card#57]
Where I live, I pay for import from the grid (consumption - €0.13) and get paid for export to the grid (production - €0.075).
These 2 rates have been added to the card config.

In the screenshot I have exported €0.31 and imported €0.43 and I have to pay €0.12.  That's fine.

The values circled in red, however, seem meaningless to me.  I do not think that they actually exist.
What does €1.11 on top mean, for example.? This is the total solar energy generated (export to grid + used by house) at the rate of 0.075.

![90890055-88b15780-e3b9-11ea-92dc-cac5dfc1f35d](https://user-images.githubusercontent.com/34303331/90891849-ac29d180-e3bc-11ea-96cc-7af72c334649.png)

**Comment by gurbyz** [comment]
You're right! This view was implemented before the energy rate parameter got split into ```energy_consumption_rate``` and ```energy_production_rate```. I will hide those numbers when both rates differ.
